Common/core/platform.h

```diff
@@ -32,36 +34,48 @@
     #endif
 
     #if TARGET_OS_SIMULATOR || TARGET_IPHONE_SIMULATOR
-        #define AGS_PLATFORM_OS_WINDOWS (0)
```

Comment:
i don't know about AGS stance on this, but in other projects i contribute too the general consensus is to not add gratuitous whitespace changes to unrelated changes.

Reply:
They are not gratuitous, I will need the additional space soon and I don't know yet when this will be merged.

Reply:
I don't quite understand though, what are these spaces for?

Reply:
I just wanted to avoid merge conflicts with the Emscripten change in case this takes a while - it may not be trivial to setup a FreeBSD VM.

Reply:
My point was, you do not have to do this in Emscripten pr either, there's no real need to adjust the lengths of all the lines if you add a new one which is slightly longer.

Reply:
I will rebase and these will not be in this PR.